src/test/unit/specific_feature_tests/dialogs.js

```diff
@@ -41,48 +41,27 @@ module("dialogs-opening_or_not", {setup: prepareUnitTestModule});
  */
 /* jshint latedef: nofunc */
 function dialogTestHelper(args) {
-    // Sinon doesn't allow wrapping of `window.open` with a spy because `open`
-    // doesn't look like a function in these browsers.
-    var sinonCantWrapWindowOpen = jQuery.browser.msie &&
-        jQuery.browser.versionNumber <= 8;
+    var wymeditor = jQuery.wymeditors(0);
+
+    // Wrap the dialog function with a spy
+    args.currentTest.spy(wymeditor, "dialog");
 
     manipulationTestHelper({
         startHtml: args.noChangeHtml,
         setCaretInSelector: args.setCaretInSelector,
-        prepareFunc: prepareFunc,
+        prepareFunc: args.prepareFunc,
         manipulationClickSelector:
             getDialogToolbarSelector(args.dialogName),
+        manipulationKeyCombo:
+            getDialogKeyCombo(args.dialogName),
         expectedResultHtml: args.noChangeHtml,
         additionalAssertionsFunc: additionalAssertionsFunc
     });
 
-    function prepareFunc(wymeditor) {
-        // Wrap the dialog function with a spy
-        args.currentTest.spy(wymeditor, "dialog");
```

Comment:
Instead of wrapping the dialog here and unwrapping it afterwards, with each `manipulationCause` run, we wrap it above, in the beginning of `dialogTestHelper`.